packages/css/src/components/standalone-link/standalone-link.scss

```diff
@@ -43,17 +43,3 @@
     color: var(--ams-standalone-link-inverse-hover-color);
   }
 }
-
-.ams-standalone-link--with-icon {
```

Comment:
This is breaking for the CSS package. If you use this class in a custom component, you should remove it. Should we keep an empty class here to remind us to mark it as a breaking change when we finally remove it?